packages/carbon-react/README.md

```diff
@@ -0,0 +1,78 @@
+# @carbon/react
+
+> React components for the Carbon Design System
+
+## Getting started
+
+To install `@carbon/react` in your project, you will need to run the following
+command using [npm](https://www.npmjs.com/):
+
+```bash
+npm install -S @carbon/react
+```
+
+If you prefer [Yarn](https://yarnpkg.com/en/), use the following command
+instead:
+
+```bash
+yarn add @carbon/react
+```
+
+## Usage
+
+**Note: this package is unstable and will change before it's 1.0 release**
+
+You can use the `@carbon/react` to bring in components, icons, and styles from
+the Carbon Design System.
+
+To include a component, you can import it:
+
+```jsx
+import { Button } from '@carbon/react';
+
+function MyComponent() {
+  return <Button>Example usage</Button>;
+}
+```
+
+To include an icon, use the `@carbon/react/icons` entrypoint and import it the
+same way you would import a component:
+
+```jsx
+import { Add } from '@carbon/react/icons';
+
+function MyComponent() {
+  return <Add />;
```

Comment:
Will users be able to specify a size? Something like `<Add size={32} />`?

Reply:
@tw15egan yes! Definitely. Will try and get some explicit docs for it, but for now we have: https://github.com/carbon-design-system/carbon/blob/main/docs/migration/11.x-carbon-icons-react.md#overview to cover size albeit indirectly

Reply:
Perfect!